Formula/weboob.rb

```diff
@@ -117,7 +117,7 @@ def install
                 "JPEG_ROOT = ('#{Formula["jpeg"].opt_prefix}/lib', '#{Formula["jpeg"].opt_prefix}/include')"
         s.gsub! "FREETYPE_ROOT = None",
                 "FREETYPE_ROOT = ('#{Formula["freetype"].opt_prefix}/lib', " \
-                                 "'#{Formula["freetype"].opt_prefix}/include')"
+                "'#{Formula["freetype"].opt_prefix}/include')"
```

Comment:
Same with the other `pillow` formulae

Reply:
Done in https://github.com/Homebrew/homebrew-core/pull/80307